Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -0,0 +1,118 @@
+"""
+Utils to interact with HDFS.
+"""
+
+import sys
+import subprocess
+import select
+import os
+
+from . import HDFSException
+
+DEVNULL = open(os.path.devnull, "w")
+
+def check_stderr(proc, timeout=None):
+    """ Check STDERR of the subprocess and kill it if there`s something.
+
+    If STDERR is closed, waits till the process ends.
+    """
+    if not isinstance(proc, subprocess.Popen):
+        raise TypeError("proc must be an instance of subprocess.Popen")
+    ready, _, _ = select.select((proc.stderr, ), (), (), timeout)
+    if ready:
+        err = proc.stderr.readline()
+        if err:
+            proc.kill()
+            raise HDFSException(err)
+    if not timeout:
+            proc.wait()
+    return proc.poll()
+
+def putfile(fname, dest):
+    """ Upload file to HDFS. """
+    cmd = ["hadoop", "fs", "-put", fname, dest]
+    try:
+        proc = subprocess.Popen(cmd,
+                                stdin =subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                stdout=DEVNULL)
+        if check_stderr(proc):
+            raise(subprocess.CalledProcessError(proc.returncode, cmd))
+    except (subprocess.CalledProcessError, OSError, HDFSException), err:
+        raise RuntimeError("(ERROR) Failed to put file to HDFS: %s\n"
+                           "Error message: %s\n" % (fname, err))
+
+def getfile(fname):
+    """ Download file from HDFS.
+
+    Check if there already is a local version of the file and remove it.
```

Comment:
Got to fix the comment, thank you.
First it was like "Function goal: download file; check the local and remove it."
But after 37c71c9 (`pyDKB/hdfs: prevent getfile() from removing local file
`) it no longer removes the local file.